minecraft-server/start-deployFTB

```diff
@@ -68,6 +68,11 @@ if [[ $startScriptCount = 0 ]]; then
   unzip -o ${srv_modpack} -d ${FTB_BASE_DIR} | awk '{printf "."} END {print ""}'
 fi
 
+if [ ! -f ./ServerStart.sh ]; then
```

Comment:
I'll have to test this out later today, but I don't think it'll work as is since 

1. The current path at this point in the script is `/data`. 
2. It also gets tricky since CurseForge modpacks create another level of directory that contains the `ServerStart.sh`, which is why I had to resort to the `find ${FTB_DIR} -name ServerStart.sh` a couple of lines down.

If you shift this past the `export FTB_DIR` line you could change the test to 

`if [ ! -f ${FTB_DIR}/${FTB_SERVER_START} ]; then`